### PR TITLE
refactor: use bootstrap handlers in go-mod-bootstrap and refactor device command application layer code

### PR DESCRIPTION
**Pull request by hahattan**
related to #1189, refactored device command application layer so we have better API to use in internal message bus subscriber.

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-sdk-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [x] I have added unit tests for the new feature or bug fix (if not, why?)
- [x] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

File: pkg/service/main.go [hunk 1]
```diff
@@ -90,8 +90,32 @@ func Main(serviceName string, serviceVersion string, proto interface{}, ctx cont
...
+	}
+
+	if !handlers.NewClientsBootstrap().BootstrapHandler(ctx, wg, startupTimer, dic) {
+		return false
+	}
+
+	return true
+}
+
+func MessageBusBootstrapHandler(ctx context.Context, wg *sync.WaitGroup, startupTimer startup.Timer, dic *di.Container) bool {
```
Comment (lenny-intel) [inline pkg/service/main.go:106]: Should this be private functions since they are only called from above.

File: internal/application/command.go [hunk 2]
```diff
@@ -30,91 +33,76 @@ import (
...
+		lc := bootstrapContainer.LoggingClientFrom(dic.Get)
+		dc := bootstrapContainer.DeviceClientFrom(dic.Get)
+		go sdkCommon.UpdateLastConnected(device.Name, lc, dc)
 	}
 
-	// check device's AdminState
-	if device.AdminState == models.Locked {
-		return res, errors.NewCommonEdgeX(errors.KindServiceLocked, fmt.Sprintf("device %s locked", device.Name), nil)
+	lc := bootstrapContainer.LoggingClientFrom(dic.Get)
+	lc.Infof("GET Device Command successfully. Device: %s, Source: %s, %s: %s", deviceName, commandName, common.CorrelationHeader, utils.FromContext(ctx, common.CorrelationHeader))
```
Comment (lenny-intel) [inline internal/application/command.go:69]:
```suggestion
	lc.Debugf("GET Device Command successfully. Device: %s, Source: %s, %s: %s", deviceName, commandName, common.CorrelationHeader, utils.FromContext(ctx, common.CorrelationHeader))
```

File: internal/application/command.go [hunk 3]
```diff
@@ -30,91 +33,76 @@ import (
...
-		}
+		err = writeDeviceResource(device, commandName, queryParams, requests, dic)
+	}
+
+	if err != nil {
+		return errors.NewCommonEdgeXWrapper(err)
 	}
+
+	lc := bootstrapContainer.LoggingClientFrom(dic.Get)
+	lc.Infof("SET Device Command successfully. Device: %s, Source: %s, %s: %s", deviceName, commandName, common.CorrelationHeader, utils.FromContext(ctx, common.CorrelationHeader))
```
Comment (lenny-intel) [inline internal/application/command.go:98]:
```suggestion
	lc.Debugf("SET Device Command successfully. Device: %s, Source: %s, %s: %s", deviceName, commandName, common.CorrelationHeader, utils.FromContext(ctx, common.CorrelationHeader))
```

**Review by lenny-intel — CHANGES_REQUESTED** [review]

**Review by lenny-intel — APPROVED** [review]
LGTM, just need to address the conflicts

**Review by lenny-intel — DISMISSED** [review]

**Comment by lenny-intel** [comment]
@hahattan , still have conflicts to resolve.

**Comment by hahattan** [comment]
recheck

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Comment by cloudxxx8** [comment]
recheck

**Pull request merged**